mantis-server/mantis-server-worker/src/main/java/io/mantisrx/server/worker/WorkerExecutionOperationsNetworkStage.java

```diff
@@ -622,7 +627,13 @@ private WorkerConsumer connectToObservableAtPreviousStages(Observable<JobSchedul
 
     @Override
     public void shutdownStage() throws IOException  {
-        logger.info("Shutdown initiated");
+        if (jobStatus != null) {
+            final Heartbeat heartbeat = heartbeatRef.get();
+            final Status status = new Status(heartbeat.getJobId(), heartbeat.getStageNumber(), heartbeat.getWorkerIndex(), heartbeat.getWorkerNumber(),
+                Status.TYPE.INFO, getWorkerStringPrefix(heartbeat.getStageNumber(), heartbeat.getWorkerIndex(), heartbeat.getWorkerNumber()) + " shutdown",
```

Comment:
nit: It would be nice to put all these message patterns in the same place.

Reply:
+1